src/app/common/components/MustGatherBtn.tsx

```diff
@@ -0,0 +1,105 @@
+import * as React from 'react';
+import { Button, Tooltip } from '@patternfly/react-core';
+import { WarningTriangleIcon } from '@patternfly/react-icons';
+import { MustGatherContext } from '@app/common/context';
+import { getMustGatherApiUrl } from '@app/queries/helpers';
+import { IMustGatherResponse } from '@app/client/types';
+import { UseQueryResult } from 'react-query';
+import { IPlan } from '@app/queries/types';
+
+interface IMustGatherBtn {
+  mg: IMustGatherResponse | undefined;
+  mustGathers: UseQueryResult<IMustGatherResponse[], Response>;
+  customName: string;
+  type: 'plan' | 'vm';
+}
+
+export const MustGatherBtn: React.FunctionComponent<IMustGatherBtn> = ({
+  mg,
+  mustGathers,
+  customName,
+  type,
+}) => {
+  const {
+    setMustGatherModalOpen,
+    setActiveMustGather,
+    mustGatherWatchList,
+    setMustGatherWatchList,
+  } = React.useContext(MustGatherContext);
+
+  return (
+    <>
```

Comment:
I don't think you need this wrapped in a `<>` fragment, you could just have the ternary be directly what you return:

```ts
return mg?.status === 'completed' && mg?.['archive-name'] ? (
  <Tooltip
  ...
) : (
  ...
);
```